circle_area.py

```diff
@@ -1,3 +1,24 @@
-# TODO: DEFINE a function using the def keyword called `area_of_circle`
-# TODO: implement the function to RETURN (NOT PRINT) the area of a circle whose radius is r
-# HINT: You'll need to import math to solve this problem correctly.
+def is_leap(year):
```

Comment:
The name of the function should be `area_of_circle`. That's likely the reason your tests are failing